### PR TITLE
IDEX-3495: show Java class structure

### PR DESCRIPTION
**Pull request by svor**

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java [hunk 1]
```diff
@@ -131,10 +132,9 @@ public static String getFQNForFile(VirtualFile file) {
                 packageName = packageName + ".";
             }
             return packageName + file.getName().substring(0, file.getName().lastIndexOf('.'));
-        } else if (file instanceof ContentNode) {
+        } else if (file instanceof ContentNode || file instanceof JarFileNode) {
```
Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java]: Looks like ContentNode and JarFileNode are similar

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/filestructure/FileStructure.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    void close();
+
+    interface ActionDelegate {
+        /**
+         * Closes window and select a region of the active element in the editor.
+         *
+         * @param member
+         *         selected member
+         */
+        void actionPerformed(Member member);
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/filestructure/FileStructure.java]: formating

File: plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/svg/file-navigation.svg [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/svg/file-navigation.svg]: Do we have a rights to use this image?

Comment (svor) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/resources/org/eclipse/che/ide/ext/java/client/svg/file-navigation.svg]: This icon was drawn by Stevan, I just move it to the other folder.

**Comment by vparfonov** [comment]
ok for me

**Pull request merged**